apps/search/documents/base.py

```diff
@@ -0,0 +1,12 @@
+from django_elasticsearch_dsl import Document as ESDocument
+from django_elasticsearch_dsl import fields
+
+
+class Document(ESDocument):
+
+    hidden = fields.BooleanField()
+    verified = fields.BooleanField()
+
+    @classmethod
+    def index_name(cls, index=None):
```

Comment:
@alashow, can you name this `get_index_name`? 
(I like the convention of `thing.property` and `thing.get_property()`)